src/module-elasticsuite-core/Model/Search/Request/RelevanceConfig/Structure/Data.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Thomas Klein <thomasklein876@gmail.com>
+ * @copyright 2020 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+declare(strict_types=1);
+
+namespace Smile\ElasticsuiteCore\Model\Search\Request\RelevanceConfig\Structure;
+
+use Magento\Framework\Config\Data\Scoped;
+
+class Data extends Scoped
```

Comment:
Please add PHPdoc for the class with the following template 
```
/**
 * Description goes here.
 *
 * @category Smile
 * @package  Smile\ElasticsuiteCore
 * @author   Thomas Klein <thomasklein876@gmail.com>
 */
```

Reply:
Will do